src/Trixi.jl

```diff
@@ -225,9 +229,10 @@ export density, pressure, density_pressure, velocity, global_mean_vars,
        equilibrium_distribution, waterheight_pressure
 export entropy, energy_total, energy_kinetic, energy_internal, energy_magnetic,
        cross_helicity,
-       enstrophy
+       enstrophy, magnetic_field, divergence_cleaning_field
 export lake_at_rest_error
 export ncomponents, eachcomponent
+export get_component
```

Comment:
Is this public API to be used?

Reply:
Yes, although this function is not currently used by the elixirs, it is useful to set source terms and boundary conditions. Is any further action required to make this public API?

Reply:
Hm, in that case one should maybe think of a different name - even when it comes with a nasty abbreviation like `get_componten_MIGLMMHD`

Reply:
Why is that? It is not clashing with any other function and it has a descriptive docstring. Other similar functions that retrieve information from the conservative variables (like `density`, `pressure`,  `waterheight_pressure`, etc) do not have such nasty abbreviations... Some others like `ncomponents` retrieve information from the equation and also do not have abbreviations.

Reply:
I could think of a more descriptive name. For example, `get_hydrodynamic_state` could be better.

Reply:
Yeah, something like that - for instance maybe `get_component_state`.
Just something that points people away from using this as a function to access the `k` th component of any vector/etc and towards the understanding of `component` of a multi-species system.

Reply:
I think both your arguments have merit. However, it seems like we already have used `get_component`/`set_component!` in other places in Trixi.jl. Thus if we start renaming stuff, we should be consistent and have an idea of the bigger picture.

To keep this minor issue from derailing an otherwise speedy review process (*padding myself and Hendrik on the shoulder*), maybe just un-export it for now and deal with it in a separate issue/PR? You can always write `using Trixi: Trixi, get_component` in an elixir if you want to have the same effect as the export.

Reply:
Hm, if we start exporting `get_component` this

https://github.com/trixi-framework/Trixi.jl/blob/a151e7452d6a9ae423d4b39eb5e828ed916ea085/src/callbacks_step/analysis_dgmulti.jl#L91-L92

should also receive a docstring to avoid confusion, right?

Reply:
Yeah, thanks guys for the speedy reviews! 🙂 
I prefer then to un-export. Done in 2fbc1d7204c436673d2af2efb76ae38073e7ec1f.